eidas-saml-parser/src/main/java/uk/gov/ida/notification/eidassaml/EidasSamlParserConfiguration.java

```diff
@@ -19,19 +27,36 @@
 
     @JsonProperty
     @Valid
-    private ReplayCheckerConfiguration replayChecker = new ReplayCheckerConfiguration();
+    private final ReplayCheckerConfiguration replayChecker = new ReplayCheckerConfiguration();
 
+    @NotNull
     @JsonProperty
+    private URI url;
+
     @Valid
     @NotNull
-    private TrustStoreBackedMetadataConfiguration connectorMetadataConfiguration;
+    @JsonProperty
+    private final JerseyClientConfiguration clientConfig = new JerseyClientConfiguration();
```

Comment:
Is the reason to have this as a field, rather than just instantiated in `buildMetatronProxy()` down below so that we have a default but can also override it if needed? If so, I think that `config.yml` for the ESP will need an update?

Reply:
Otherwise I think `buildMetatronProxy()` could be defined somewhere else, like the Application class rather than in here, as all it really needs is the enviroment.

Reply:
It does need the configuration too but that can be supplied in the Application class so I've moved it.  We do need the metatronUri as a config value somehow.